### DISCUSSION TITLE
incompatible access points not visisible

### DISCUSSION THREAD

**Discussion by stlorenz** [OpenCCU/OpenCCU#2578] [🇺🇸 Q&A] [answered]
Hi, I would like to add a HMIP-HAP to my RasperryMatic installation on a RPi3. 
I am using the RPI-RF-MOD and it works with normal HM devices. 
CUxD is installed. and works as expected. 

RaspberryMatic is the latest version. 

But the Button does not appear. 

What am I doing wrong?

**Comment by Baxxy13** [comment, marked as answer]
Der Button wurde mit 3.69.7 (ca. April 2023) entfernt.
Ein HAP wird ähnlich wie andere IP-Geräte angelernt.
- IP-Anlernmodus (mit Internet) starten
- am blau leuchtenden HAP 1x auf die Taste drücken, die Farbe sollte zu türkis wechseln und kurz darauf der HAP im Posteingang erscheinen

**Reply by stlorenz** [reply]
Danke.
Funktioniert wie erwartet. Wenn ich es schaffe, mache ich morgen ein Update der Doku.

**Reply by stlorenz** [reply]
Update erfolgt, ich konnte allerdings in den release notes allerdings keinen Hinweis auf diese Änderung finden.

**Reply by Baxxy13** [reply]
Ich hatte die Info aus den Release-Notes zur CCU3 Firmware. (3.69.6)

>[HMCCU-1071] Der Button „Access Points mit inkompatibler FW” (Einstellungen → Systemsteuerung)
wurde entfernt.
